tools/modules/software/module_filebrowser.sh

```
@@ -0,0 +1,92 @@
+	["module_filebrowser,doc_link"]="https://filebrowser.org/"
+	["module_filebrowser,group"]="Utilities"
+	["module_filebrowser,port"]="8095"
+	["module_filebrowser,arch"]="x86-64 arm64"
+)
+
+function module_filebrowser () {
+	local title="filebrowser"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/filebrowser?( |$)/{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)